Resources/Prototypes/GameRules/second_life.yml

```diff
@@ -0,0 +1,115 @@
+- type: entity
+  id: LifeTourists
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    weight: 10
+    startDelay: 10
+    duration: 120
+    earliestStart: 20
+    reoccurrenceDelay: 20
+  - type: SecondLifeRule
+    announce: second-life-tourist-announce
+    description: second-life-tourist
+    roles:
+      Passenger: 15
+
+- type: entity
+  id: LifeSecurity
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+    - type: StationEvent
+      weight: 0.5
+      startDelay: 10
+      duration: 60
+      earliestStart: 20
+      reoccurrenceDelay: 60
+    - type: SecondLifeRule
+      announce: second-life-security-announce
+      description: second-life-security
+      roles:
+        Passenger: 3
+        SecurityOfficer: 5
```

Comment:
imo all of the rules that give out 3 passenger slots should be 2 instead. 5 seccies is also too much, i'd go with 2 or 3